play-services-core/src/main/AndroidManifest.xml

```diff
@@ -756,6 +756,18 @@
             </intent-filter>
         </service>
 
+        <!-- backup -->
+
+        <activity
+            android:name="org.microg.gms.backup.component.BackupSettingsCollapsingActivity"
```

Comment:
Can you rename to `org.microg.gms.backup.BackupSettingsActivity` please? There are not "component"s in backup and there's nothing "Collapsing" about that activity.

Reply:
> Can you rename to `org.microg.gms.backup.BackupSettingsActivity` please? There are not "component"s in backup and there's nothing "Collapsing" about that activity.

Okey